zulipterminal/core.py

```diff
@@ -441,6 +442,31 @@ def stream_muting_confirmation_popup(
         mute_this_stream = partial(self.model.toggle_stream_muted_status, stream_id)
         self.loop.widget = PopUpConfirmationView(self, question, mute_this_stream)
 
+    def copy_to_clipboard(self, text: str, text_category: str) -> None:
+        try:
+            pyperclip.copy(text)
+            clipboard_text = pyperclip.paste()
+            if clipboard_text == text:
+                self.report_success(f"{text_category} copied successfully")
+            else:
+                self.report_warning(
+                    f"{text_category} copied, but the clipboard text does not match"
```

Comment:
This might happen if there is an internal issue with the `pyperclip` module. Neil also explained other possible trigger conditions in [this](https://github.com/zulip/zulip-terminal/pull/880#discussion_r682727216) comment.